modules/Hoot/hoot.js

```diff
@@ -26,7 +30,7 @@ class Hoot {
         this.events       = new EventManager();
 
         this.config = {
-            urlroot: 'http://54.165.118.78:8080/hoot-services/osm',
+            urlroot: 'http://localhost:8080/hoot-services/osm',
```

Comment:
Will this need updating per deployed environment?

Reply:
this should be fine in all environments, including testing

Reply:
I don't see how this will work in production, when Hoot is not running or proxied locally.